docs/learn/bitesize-invaders.md

```diff
@@ -0,0 +1,128 @@
+
+```csharp reference
+https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/master/Basic/Invaders/Assets/Scripts/PlayerControl.cs#L157-L173
+```
+
+## Shared Start/Round Timer with Client-Side Prediction
+
+Games commonly have timers to display in the UI such as Start Timer, Round Timer, and Cooldowns. Invaders also have a shared timer to ensure all players start the game at the same time. Otherwise, players with higher-end devices and better network access may have an unfair advantage by loading scenes and maps faster.
+
+When you implement this kind of timer, usually you would use a `NetworkVariableFloat` to replicate and display the exact time value across all clients. To improve performance, you do not need to replicate that float every Network Tick to the Clients, which would only waste network bandwidth and some minimal CPU resources.
+
+You have two options:
+
+* The best solution is to sync the remaining value of the timer when a new client joins. For the remaining time, clients can locally predict what the next value of that timer is going to be. This method ensures the server does not need to send the value of that timer every Network Update tick since you know what the approximated value will be. There is a minimal overhead of keeping an additional float member variable that will be kept updated, as the clients cannot modify the `NetworkVariable` directly.
+* A fair solution is to set the `SendTickRate` of that timer `NetworkVariableFloat`, so that the server only sends an update once every second, without any additional work.
+
+In Invaders and the current state of Unity MLAPI, there is a drawback to implement such a pattern. If you set `NetworkVariableFloat` `SendTickRate` to '-1' (which means "do not send any updates anymore about this NetworkVariable to the clients"), it will not sync up the current timer value with the clients that just joined. It will never catch up with the server, which means you need to write more code to deal with this.
+
+As a workaround, you have to wait for all clients to be connected before setting the `SendTickRate` to '-1' for `m_ReplicatedTimeRemaining` `NetworkVariableFloat`. The code that checks if you need to set this is inside the `ShouldStartCountDown` method in *InvadersGame.cs*. 
+
+:::important
+As soon as MLAPI evolves and fixes this problem, you can just set `SendTickRate` in the `NetworkStart` and most of the code inside `ShouldStartCountDown` will become obsolete.
+:::
+
+### Start the game timer
```

Comment:
@Cosmin-B  Added a heading and short intro from your content for the start of timer.

Reply:
Please replace: "The server only needs to be contacted once to check if the game has started (m_HasGameStared is true) and the m_TimeRemaining amount, recieved by ShouldStartCountDown. When met, it predictively calculates and updates the gameTimer reducing the remaining time on the client-side for all players. When m_TimeRemaining reaches 0.0, the timer is up."

With: "When all the players are connected, we will get one last update for **m_TimeRemaining**, so that it will sync up with the server timer on line **151**, then, as long as the game did not start, or the **m_TimeRemaining** did not reach 0.0, we predictively set the **m_TimeRemaining** and update the UI with the result.